psql.sh

```diff
@@ -3,12 +3,13 @@
 # Tip: More info on psql - https://tomcam.github.io/postgres/
 
 POSTGRES_CONTAINER="managed-gitops-postgres"
+export POSTGRESQL_DATABASE="postgres"
```

Comment:
As above, if the environment variable is already set by the user calling script, we should use the existing value, otherwise we should default to `postgres`